### PR TITLE
[MinGW][cmake] Add cmake libs in mingw

### PR DESCRIPTION
**Pull request by talregev**
Add cmake libs in mingw.

### CODE REVIEW

**Comment by talregev** [comment]
Ready for review.

File: .github/workflows/mingw.yml [hunk 1]
```diff
@@ -18,7 +18,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        type: [libs-mingw, apps-mingw, apps-mingw-vcpkg]
+        type: [libs-mingw, apps-mingw, apps-mingw-vcpkg, libs-cmake]
```
Comment (AenBleidd) [inline .github/workflows/mingw.yml:21]: Better to rename to 'libs-mingw-cmake'

File: mingw/bootstrap_vcpkg_cmake.sh [hunk 2]
```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
+set -e
+
+OS_NAME="mingw"
```
Comment (AenBleidd) [inline mingw/bootstrap_vcpkg_cmake.sh:4]: 'mingw' is not an OS, maybe better to rename 'OS_NAME' to 'TARGET_PLATFORM_NAME' or smth similar?

**Review by AenBleidd — CHANGES_REQUESTED** [review]

File: mingw/bootstrap_vcpkg_cmake.sh (hunk 2 above)
Comment (talregev) [reply mingw/bootstrap_vcpkg_cmake.sh:4]: plt_name

Comment (talregev) [reply mingw/bootstrap_vcpkg_cmake.sh:4]: `PLATFORM_NAME`

Comment (talregev) [reply mingw/bootstrap_vcpkg_cmake.sh:4]: done.

File: .github/workflows/mingw.yml (hunk 1 above)
Comment (talregev) [reply .github/workflows/mingw.yml]: done.

File: mingw/bootstrap_vcpkg_cmake.sh (hunk 2 above)
Comment (AenBleidd) [reply mingw/bootstrap_vcpkg_cmake.sh:4]: You changed this in ci_configure_libs_cmake.sh but not in this file. Please fix it here also

**Review by AenBleidd — CHANGES_REQUESTED** [review]

File: mingw/bootstrap_vcpkg_cmake.sh (hunk 2 above)
Comment (talregev) [reply mingw/bootstrap_vcpkg_cmake.sh]: done.

**Review by AenBleidd — APPROVED** [review]
LGTM

**Pull request merged**